projects/demo/src/app/demo/pages/container-page/container-page.component.ts

```diff
@@ -19,6 +19,15 @@ export class ContainerPageComponent {
     </div>
 </div>`;
 
+    compactWithFooterExample = `<div class="pa-scrollable-container">
+    <div class="pa-main-container-tight">
+        <div class="pa-child-container-compact">
+            <article class="pa-scrollable-container-with-fixed-footer"></article>
```

Comment:
ahhhh! I thought this `with-footer` class would be at `pa-scrollable-container` level!
so probably better to name it `pa-content-with-fixed-footer`